synapsis/src/main/resources/data.sql

```diff
@@ -1,10 +1,10 @@
--- Password is 1234
-INSERT INTO app_user (name, password, email, role)
-VALUES ('Joe User', '$2a$12$ShsRhBqy8y9ep/oxB5ury.7cxmcGjt.BQA4i6dhp/RUva/hS7DjHm', 'joeuser@mail.com', 'CANDIDATE');
+-- Password is 12345678
+INSERT INTO app_user (name, password, email, role, auth_provider)
```

Comment:
Should we maybe create a dummy SSO user to add in here? (As in actually create a test google profile)